cli/crates/cli/tests/utils/macros.rs

```diff
@@ -2,22 +2,24 @@ pub use json_dotpath::DotPaths;
 
 #[macro_export]
 macro_rules! dot_get_opt {
-    ($item: ident, $dotpath: literal, $ty: ty) => {{
+    ($item: expr, $dotpath: expr, $ty: ty) => {{
```

Comment:
Why did these change?
We originally debated allowing an expression but there wasn't consensus, plus we want to keep parity with the API version.

We could discuss this again

Reply:
Because literal doesn't work with the `format!("{prefix}.edges...")` that I'm using in my loop to test nested & root pagination. It stays consistent in terms of usage with the API, a literal is still an expression.

If I remove it, I have to unroll my loop. So I'm not sure to see any benefit to not support expression currently. ¯\\_(ツ)_/¯

Reply:
I see, ok let's change it then